docs/p12-limitation.rst

```diff
@@ -0,0 +1,272 @@
+  issuer=/C=CN/ST=BJ/L=BJ/O=ABC/OU=IT/CN=abc.com/emailAddress=ask@abc.com
+  -----BEGIN CERTIFICATE-----
+  MIICbjCCAdegAwIBAgIJAMPIx273oielMA0GCSqGSIb3DQEBCwUAMHAxCzAJBgNV
+  BAYTAkNOMQswCQYDVQQIDAJCSjELMAkGA1UEBwwCQkoxDDAKBgNVBAoMA0FCQzEL
+  MAkGA1UECwwCSVQxEDAOBgNVBAMMB2FiYy5jb20xGjAYBgkqhkiG9w0BCQEWC2Fz
+  a0BhYmMuY29tMB4XDTE4MTAxNjA0MzIyNFoXDTE4MTExNTA0MzIyNFowcDELMAkG
+  A1UEBhMCQ04xCzAJBgNVBAgMAkJKMQswCQYDVQQHDAJCSjEMMAoGA1UECgwDWFla
+  MQswCQYDVQQLDAJJVDEQMA4GA1UEAwwHeHl6LmNvbTEaMBgGCSqGSIb3DQEJARYL
+  YXNrQHh5ei5jb20wgZ8wDQYJKoZIhvcNAQEBBQADgY0AMIGJAoGBAOX8VIM9O4wK
+  QT5KPcs0dVvCzJf944J3ssl+Gaa5WLyuRmYyme5YTST0CjWqDnN8ROR8OgNZi+qF
+  TJHrfk1txlWh91SPV4msKh3c5ZJp9LT/kMGnlgd8blu6FGmFZkBK6tTx7gO5LPME
+  ATvyKi1yYis0pup3AyCQgoJgjwPUWrjdAgMBAAGjEDAOMAwGA1UdEwQFMAMBAf8w
+  DQYJKoZIhvcNAQELBQADgYEAFoTBXkSm9sT1Bd3kWudqYFSQCi+xAUZ2OvDwUnxO
+  m3nO7+U9xX86ULBnYfmeT9ZohXQc44cG630ynBNOfT64/d5yJGbH7+ri/p9SkMXy
+  BVbxQHUHJjA07oyw/gJaFcOkMiSYaRObALxvaXqKFLogS0vaYb+c7P2lfkEtjOvQ
+  07Y=
+  -----END CERTIFICATE-----
+
+  # use openssl command to extract unencrypted private key
+  openssl pkcs12 -nocerts -nodes -in certificate.pfx
+  Enter Import Password: your_password
+  MAC verified OK
+  Bag Attributes
+      localKeyID: 03 A7 84 BE 07 AF 22 08 E1 AD 2C 22 2F F9 16 28 D1 BE 90 F7
+  Key Attributes: <No Attributes>
+  -----BEGIN PRIVATE KEY-----
+  MIICdgIBADANBgkqhkiG9w0BAQEFAASCAmAwggJcAgEAAoGBAMUlisLamDXh1Hen
+  BLGjw6kAcY1tABNd1KxI8amjw6q8vo5KpMRv7TI795pGuXiLg3jMMtUzU2t3w6o9
+  02cjnv5LtCWMYwdxajDgAULbjNjuQU2sWVaO61ECyr+gXUbQAuQgaXEXy+JYxnYO
+  rkcyepsmoHPS24dDYkiqU7sHT4evAgMBAAECgYAh/IDZapnG4DGilvZmMVcvsVXi
+  DaJvtENPmBuhJQw8SOGI4jFZc/ouKMxcmkirEVIO5xT0UC7XXgQrOxFQvIFSBAFu
+  mnwu1xo0YuWCYevgvto+W8f9KWEfF0fm2B5TaEWGQuvkQO6NPBBhynPxwScGrP8W
+  0H3epH4GBFfuyMmYoQJBAOHu++fajw/EwRYwoO9jdq5n+8KLF7UUbCbLicssTlYp
+  xYVcB/eiri5GCVMpAfVwPu51Iq9DwM8vhaRaIeeXBLkCQQDfYduP5UvZq1bt4+it
+  KgDMuCpOFgHpsbkHKYjDiUKlyQUJTpilMhUQaEYOuuBtSUPiHbnVIeDGevEDNjUu
+  k4unAkEAijXi/euSlqfbIpLu8GKdKIXN2i2yfACZRIZArfxVk5oMDc/Fw/RNr1sC
+  u0+btiRLKx6W8DZ3TYcZrvfAQ+b7aQJAKiOgsq1zuJUs6eWDgB8bVYMsX9yg8b/Y
+  ArTgGtrq2rFoJNBR1V/0P0scZCz2p0cZ6i7nXoBskaShkJSKRcYJowJAOiqT4v1W
+  UyJbnba983AFR+KJGuFAN1QGr6RoQmWpGv+mhnLcxFUtCiZvtFLIm5cbjdJrO8S1
+  t2Ee3jpA/oUeaQ==
+  -----END PRIVATE KEY-----
+
+- Compare each output of ``openssl`` command to the original certificates ``server.pem`` and ``int-ca.pem``, the server and intermediate certificate can be easily found. The private key context is different from the original one, since ``openssl`` command regenerates it when extracting the PKCS 12 file.
+- When creating SSL certificate in F5 LBaaS dashboard:
+
+  * Fill the server certificate into the ``Certificate`` box.
+  * Fill the intermediate certificate into the ''Certificate Chain'' box.
+  * Fill the private key into the ``Private Key`` box.
+  * The private key is not encrypted, when extracting PKCS 12 file, thus, leave the ``Passphrase`` blank.
```

Comment:
``Leave the ``Passphrase`` field blank.``

Reply:
done